tools/qontract_cli.py

```diff
@@ -2249,6 +2249,109 @@ def sre_checkpoints(ctx: click.Context) -> None:
     print_output(ctx.obj["options"], checkpoints_data, columns)
 
 
+@get.command()
+@click.pass_context
+@click.option(
+    "--repo-name",
+    help="Specifies the gitlab repo name",
+    type=str,
+)
+@click.option(
+    "--ignore-awaiting-approval",
+    help="Ignore MR awaiting approval",
+    default=False,
+    type=bool,
+)
+def gitlab_repo_merge_request(
```

Comment:
do we run this command in pipeline, it will increase gitlab requests a lot and not shown in metrics

Reply:
that report can be helpful for whole repo mrs status, just to answer "Why hasn’t my MR been merged?", would it better to let bot left comment on MR on status or apply label? Do it as part of housekeeping run.